### PR TITLE
fix timezone issue on testing.

### PR DESCRIPTION
**Pull request by songsungkyun**
some tests with datetime are failed at South Korea.
so add some code with timezone and it's resolved.

### CODE REVIEW

File: src/main/resources/querydsl_checks_base.xml [hunk 1]
```diff
@@ -8,7 +8,9 @@
   <property name="fileExtensions" value="java, properties, xml"/>
 
   <!-- maximum file length 2000 lines -->
-  <module name="FileLength"/>
+  <module name="FileLength">
+      <property name="max" value="2500"/>
```
Comment (Shredder121) [inline src/main/resources/querydsl_checks_base.xml]: This is only overridden on a per-file basis.

**Comment by timowest** [comment]
@songsungkyun Which tests failed?

**Comment by songsungkyun** [comment]
This is a part of failed.

```
asDate_returns_a_corresponding_DateExpression_for_a_given_Constant(com.querydsl.core.types.dsl.ExpressionsTest)  Time elapsed: 0.008 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<year(Thu Jan 01 0[0:00:00 UTC] 1970)> but was:<year(Thu Jan 01 0[9:00:00 PWT] 1970)>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.querydsl.core.types.dsl.ExpressionsTest.asDate_returns_a_corresponding_DateExpression_for_a_given_Constant(ExpressionsTest.java:312)

asTime_returns_a_corresponding_TimeExpression_for_a_given_Expression(com.querydsl.core.types.dsl.ExpressionsTest)  Time elapsed: 0 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<hour(Thu Jan 01 0[0:00:00 UTC] 1970)> but was:<hour(Thu Jan 01 0[9:00:00 PWT] 1970)>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.querydsl.core.types.dsl.ExpressionsTest.asTime_returns_a_corresponding_TimeExpression_for_a_given_Expression(ExpressionsTest.java:328)

asDateTime_returns_a_corresponding_DateTimeExpression_for_a_given_Constant(com.querydsl.core.types.dsl.ExpressionsTest)  Time elapsed: 0.001 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<min(Thu Jan 01 0[0:00:00 UTC] 1970)> but was:<min(Thu Jan 01 0[9:00:00 PWT] 1970)>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.querydsl.core.types.dsl.ExpressionsTest.asDateTime_returns_a_corresponding_DateTimeExpression_for_a_given_Constant(ExpressionsTest.java:323)

asDateTime_returns_a_corresponding_DateTimeExpression_for_a_given_Expression(com.querydsl.core.types.dsl.ExpressionsTest)  Time elapsed: 0 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<min(Thu Jan 01 0[0:00:00 UTC] 1970)> but was:<min(Thu Jan 01 0[9:00:00 PWT] 1970)>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.querydsl.core.types.dsl.ExpressionsTest.asDateTime_returns_a_corresponding_DateTimeExpression_for_a_given_Expression(ExpressionsTest.java:317)

asTime_returns_a_corresponding_TimeExpression_for_a_given_Constant(com.querydsl.core.types.dsl.ExpressionsTest)  Time elapsed: 0 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<hour(Thu Jan 01 0[0:00:00 UTC] 1970)> but was:<hour(Thu Jan 01 0[9:00:00 PWT] 1970)>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.querydsl.core.types.dsl.ExpressionsTest.asTime_returns_a_corresponding_TimeExpression_for_a_given_Constant(ExpressionsTest.java:334)

asDate_returns_a_corresponding_DateExpression_for_a_given_Expression(com.querydsl.core.types.dsl.ExpressionsTest)  Time elapsed: 0.001 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<year(Thu Jan 01 0[0:00:00 UTC] 1970)> but was:<year(Thu Jan 01 0[9:00:00 PWT] 1970)>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.querydsl.core.types.dsl.ExpressionsTest.asDate_returns_a_corresponding_DateExpression_for_a_given_Expression(ExpressionsTest.java:306)
````

File: src/main/resources/querydsl_checks_base.xml (hunk 1 above)
Comment (songsungkyun) [inline src/main/resources/querydsl_checks_base.xml]: rollbacked that.
I misunderstood it.

**Comment by johnktims** [comment]
Other than maybe squashing some of the commits, I think this change is fine.

File: querydsl-sql/src/test/java/com/querydsl/sql/ArrayTypesTest.java [hunk 2]
```diff
@@ -13,6 +14,9 @@
 
     @Before
     public void setUp() {
+        System.setProperty("user.timezone", "UTC");
```
Comment (timowest) [inline querydsl-sql/src/test/java/com/querydsl/sql/ArrayTypesTest.java]: what failures do you have with this test?

File: querydsl-core/src/test/java/com/querydsl/core/types/dsl/ExpressionsTest.java [hunk 3]
```diff
@@ -36,6 +38,12 @@
         TEST;
     }
 
+    @Before
+    public void before() {
+        System.setProperty("user.timezone", "UTC");
```
Comment (timowest) [inline querydsl-core/src/test/java/com/querydsl/core/types/dsl/ExpressionsTest.java]: also reset in after block

File: querydsl-sql/src/test/java/com/querydsl/sql/ArrayTypesTest.java (hunk 2 above)
Comment (songsungkyun) [inline querydsl-sql/src/test/java/com/querydsl/sql/ArrayTypesTest.java]: @timowest 
weird.
I remembered it was failed with testing on querydls-sql module.
but it's gone now.

File: querydsl-core/src/test/java/com/querydsl/core/types/dsl/ExpressionsTest.java (hunk 3 above)
Comment (songsungkyun) [inline querydsl-core/src/test/java/com/querydsl/core/types/dsl/ExpressionsTest.java]: @timowest 
Yeah. I added setUp and tearDown.
I rewrote small bit of code.

**Comment by timowest** [comment]
Could you squash the commits? Then it would be fine to merge.

**Comment by songsungkyun** [comment]
moved to https://github.com/querydsl/querydsl/pull/1775

**Comment by timowest** [comment]
you can also force push the result of the rebase.

**Comment by Shredder121** [comment]
Yes, as long as you move the tip of the branch the pull request points to (`songsungkyun:timezone`, so your `timezone` branch) the pull request reflects that change.

**Pull request closed without merging**